.github/workflows/3rd-party-tests.yml

```diff
@@ -110,7 +110,7 @@ jobs:
         env:
           URL: postgresql+psycopg://postgres:password@127.0.0.1/test
         working-directory: sa_home/sa
-        run: pytest -n 2 -q --dburi $URL --backend-only --dropfirst --color=yes
```

Comment:
Does this run only async instead of sync? Don't we want to run both sync and async?

Reply:
No, it runs both. It's not overly clear, but the `dbdriver` option is used to add additional drivers to run with the provided uri.

If it's clearer, both versions could be added as `dbdriver` since the testing framework of sqlalchemy will deduplicate them

Reply:
Ok, thank you for confirming, and indeed the test run shows almost twice the number of tests. :+1:

Reply:
Yes, it basically creates a copy of every "backend" test for each driver selected